src/bin/wasmer.rs

```diff
@@ -40,6 +40,13 @@ use wasmer_runtime_core::{
 #[cfg(feature = "wasi")]
 use wasmer_wasi;
 
+#[cfg(all(
+    not(feature = "backend-cranelift"),
+    not(feature = "backend-llvm"),
+    not(feature = "backend-singlepass")
+))]
```

Comment:
I think this is a bit easier to read:

```suggestion
#[cfg(not(any(
    feature = "backend-cranelift",
    feature = "backend-llvm",
    feature = "backend-singlepass"
)))]
```